doc/tutorials/acrn-dm_QoS.rst

```diff
@@ -0,0 +1,121 @@
+.. _acrn-dm_qos:
+
+Enable QoS based on runC container
+##################################
+ACRN supports Device-Model QoS based on runC container to control
+the SOS resources (CPU, Storage, Memory, Network) by modifying the runC configuration file.
```

Comment:
Should this be UOS instead of SOS? You talk about UOS later in this doc.

Reply:
This is the SOS resources used by ACRN-dm process.